_posts/2019-07-29-plonk-stack.md

```diff
@@ -0,0 +1,211 @@
+
+* What comes after OpenFaaS?
+    In late 2017 I started to design a distribution of OpenFaaS that shipped as a whole package including HTTPS, multi-user support, authz and CI/CD with GitHub and GitLab.
+
+    <blockquote class="twitter-tweet" data-theme="light"><p lang="en" dir="ltr">&quot;OpenFaaS Cloud - Community Cluster&quot;<br><br>A SaaS solution built on OpenFaaS.<br><br>- Free sub-domain and endpoints<br>- TLS by default<br>- Login with GitHub<br>- No complex API or CLI needed, just push to git.<br><br>Apply for access today. 🏆<a href="https://t.co/IGuZGZiPT3">https://t.co/IGuZGZiPT3</a> <a href="https://twitter.com/hashtag/gitops?src=hash&amp;ref_src=twsrc%5Etfw">#gitops</a> <a href="https://twitter.com/hashtag/faasfriday?src=hash&amp;ref_src=twsrc%5Etfw">#faasfriday</a> <a href="https://t.co/pqIMzYsMbI">pic.twitter.com/pqIMzYsMbI</a></p>&mdash; OpenFaaS (@openfaas) <a href="https://twitter.com/openfaas/status/1144621477129379842?ref_src=twsrc%5Etfw">June 28, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+    
+    It's called OpenFaaS Cloud and is free and open-source. You can host your own using the [ofc-bootstrap tool](https://github.com/openfaas-incubator/ofc-bootstrap) or [request free access to the Community Cluster](https://github.com/openfaas/community-cluster/).
+
+    OpenFaaS Cloud is great for teams and for multi-user setups. You can even use it to host a SaaS, like we are doing with the Community Cluster.
+
+* But are you ready for Serverless?
+
+    I often hear people say to me "we are not ready for Serverless". If you're ready for Kubernetes, then OpenFaaS gives you a lower barrier to entry, a stream-lined developer experience and a passionate community of real users. For me Serverless describes an approach to architecture, rather than some new magic that users can be "ready for", or not.
+
+    Here's an example of how OpenFaaS can save you code on packaging.
+
+    <blockquote class="twitter-tweet" data-conversation="none"><p lang="en" dir="ltr">What does it get you? <a href="https://t.co/GrQCrYtKNr">pic.twitter.com/GrQCrYtKNr</a></p>&mdash; Alex Ellis (@alexellisuk) <a href="https://twitter.com/alexellisuk/status/1154385354755125248?ref_src=twsrc%5Etfw">July 25, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+* Summing up
+
+    It's hard to sum up 3-years of R&D, community interaction, events, blog posts, tutorials, and workshops, but the end result looks something like: PLONK. I hope you'll kick the tires with OpenFaaS if you haven't already, or if you tried it some time ago, will come back and see how much it's improved.
```

Comment:
Id love it if we could stay closer to our english roots with the spelling of tyres

Reply:
I always use "tires", but I am OK with changing it.